index.html

```diff
@@ -202,58 +225,59 @@ <h3>Languages</h3>
               <li><img src="images/html-language.png"  alt="Languages"></li>  
               <li><img src="images/css-language.png"  alt="Languages"></li>  
             </ul>     
-
+            <ul class="ul-categories1">
+              <li><img src="images/js-vertical.png"  alt="Languages"></li>  
+              <li><img src="images/html-vertical.png"  alt="Languages"></li>  
+              <li><img src="images/css-vertical.png"  alt="Languages"></li>  
+            </ul>     
             <div class="about-items">
               <h3>Frameworks</h3>
               <img src="images/greater-than.png"  alt="open">      
             </div>
-            <ul class="ul-categories">
-                  <li></li>
-            </ul>
 
-              <div class="about-items">
-                <h3>Skills</h3>             
-                <img src="images/greater-than.png"  alt="open">  
-              </div>
+            <div class="about-items">
+              <h3>Skills</h3>             
+              <img src="images/greater-than.png"  alt="open">  
+            </div>
             <ul class="ul-categories">
                 <li></li>
-            </ul>
-              
+            </ul>              
           </div>
-        </section>
-
-        <section class="contactForm-section ">
-          
 
-          <h2>Contact me</h2>
-          <p>If you have an application you<br>
-              are interested in developing, a<br>
-              feature that you need built or a<br>
-              project that needs coding. I’d<br>
-              love to help with it</p> 
-          <form  class="contact-form" action="https://formspree.io/f/mgedjayo" method="post">
-            <ul class="contact-input-form">
-              <li>              
-                <input class="input" type="text" name="full_name" placeholder="Full name"
-                  maxlength="30" required />
-              </li>
-              <li>
-                <input class="input" type="email" name="email" placeholder="Email address"
-                required /> 
-              </li>
-              <li>
-                <textarea placeholder="Write your message here"
-                 class="txt" name="contact_text" 
-                 cols="30" rows="5"  maxlength="500" required
-                 ></textarea>
-              </li>
-              <li>
-                <button type="submit" class="contact-button">
-                Get in Touch</button>
-              </li>              
-            </ul>                
-          </form>
+        </section>
 
+        <section class="contactform-section">        
+          <div class="contact-info">
+            <h2>Contact me</h2>
+            <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or a
+                project that needs coding. I'd
+                love to help with it</p> 
```

Comment:
- [OPTIONAL] I noticed the background color in the contact section looks different from the [Figma] design, so I think it would be better to use the same background. 👌

| Your Design  |  Figma Design |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/178130117-78ea859e-0156-4a30-bc4c-210b6126829a.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/178130107-28a703df-3e4a-49bf-8f8c-c59a7d3623f9.PNG" width="500" /> |